doc/BGP/Bgpcfgd-dyn-peer-modification-support.md

```diff
@@ -0,0 +1,144 @@
+# Bgpcfgd Dynamic Peer moditication support
+# High Level Design Document
+### Rev 0.1
+
+# Table of Contents
+  * [Revision](#revision)
+
+  * [About this Manual](#about-this-manual)
+
+  * [Use case](#use-case)
+
+  * [Definitions/Abbreviation](#definitionsabbreviation)
+ 
+  * [1 Requirements Overview](#1-requirements-overview)
+    * [1.1 Functional requirements](#11-functional-requirements)
+    * [1.2 Scalability requirements](#12-scalability-requirements)
+  * [2 Modules Design](#2-modules-design)
+    * [2.1 Config DB](#21-config-db)
+    * [2.2 State DB](#22-state-db)
```

Comment:
could you clarify what's the usage of state db here?

Reply:
Added, thanks